src/index.ts

```diff
@@ -18,8 +19,11 @@ const cli = new Cli({
 
 const commandsPath = `${__dirname}/commands`
 for (const commandFolder of fs.readdirSync(commandsPath)) {
-  // tslint:disable-next-line: no-var-requires
-  require(`${commandsPath}/${commandFolder}`).forEach(cli.register.bind(cli))
+  const commandPath = `${commandsPath}/${commandFolder}`
+  if (fs.statSync(commandPath).isDirectory()) {
+    // tslint:disable-next-line: no-var-requires
+    require(commandPath).forEach((command: CommandClass) => cli.register(command))
+  }
```

Comment:
why is this now needed?

Reply:
I had an issue with Mac OS storing the `.DS_Store` file in the command folder. `require('command/.DS_Store')` causes an error when I wanted to test using `yarn launch` :)

Reply:
Oh I see. Makes sense then.